geometry_msgs/Point      pos
geometry_msgs/Quaternion ori_quaternion
geometry_msgs/Vector3    ori_euler
